build/dsc.rake

```diff
@@ -56,11 +56,43 @@ eod
       if !is_custom_resource
         puts "Downloading and Importing #{item_name}"
         cmd = "git clone #{dsc_repo} #{dsc_resources_path_tmp} && " +
-          "cd #{dsc_resources_path_tmp} && "
-        cmd += "git checkout #{ENV['DSC_REF']} &&" if ENV['DSC_REF']
-        cmd += "git submodule update --init --recursive"
+          "cd #{dsc_resources_path_tmp}"
+          #"cd #{dsc_resources_path_tmp} && "
+        #cmd += "git checkout #{ENV['DSC_REF']} &&" if ENV['DSC_REF']
+        #cmd += "git submodule update --init --recursive"
```

Comment:
Th recursive submodule init and update is removed in favor of handling the source repos directly.